src/main/java/seedu/intern/model/applicant/GraduationYearMonth.java

```diff
@@ -36,15 +36,17 @@ public GraduationYearMonth(String graduationYearMonth) {
      * Returns true if a given string is a valid grade name.
      */
     public static boolean isValidGraduationYearMonth(String test) {
-        int holder;
+        int monthHolder;
+        int yearHolder;
         if (test.matches(VALIDATION_REGEX)) {
-            holder = Integer.parseInt(test.split("/")[0]);
+            monthHolder = Integer.parseInt(test.split("/")[0]);
+            yearHolder = Integer.parseInt(test.split("/")[1]);
         } else {
             return false;
         }
 
-        // month in between 1-12 inclusive
-        return test.matches(VALIDATION_REGEX) && holder >= 1 && holder <= 12;
+        // month in between 1-12 inclusive and year lower bound of 2020
+        return test.matches(VALIDATION_REGEX) && monthHolder >= 1 && monthHolder <= 12 && yearHolder >= 1980;
```

Comment:
I feel the .now() method works better in our case or a higher lower bound should be set than 1980 as internship application is useful with a short period of usage. Either way we should just specify in UG/DG. What do you think